src/utils/validation.ts

```diff
@@ -76,3 +89,24 @@ export const validateBlockDirectory = (): void => {
         process.exit(1);
     }
 };
+
+export const integrationIdFromName = (
+    integrationName: string | undefined
+): number | undefined => {
+    if (!integrationName) {
+        return undefined;
+    }
+
+    const integration = INTEGRATIONS[integrationName!.toLowerCase()];
+    if (!integration) {
+        logError(
+            `\n"${integrationName}" is not a valid integration. The options are:\n${Object.keys(
+                INTEGRATIONS
+            )
+                .map((x) => `\t- ${x}`)
+                .join("\n")}`
```

Comment:
one more place for that integration-listing function.

Reply:
This is actually different enough from the ones in `index` that I think it makes sense to be separate, but I pulled it out into a variable for more clarity. Sometimes I can get a bit carried away with my one-liners :P